extensions/amp-access/0.1/login-dialog.js

```diff
@@ -267,7 +267,7 @@ export class WebLoginDialog {
     }
     dev().fine(TAG, 'Login done: ', result, opt_error);
     if (opt_error) {
-      this.reject_(opt_error);
+      this.reject_(user().createError('login failed', opt_error));
```

Comment:
Those both are `dev` errors. The point of `createError` at this time is to override the dev/user flag, though we could certainly have one (different one, not part of `dev` or `user`) that preserves it.